src/commands/package.command.ts

```diff
@@ -48,7 +48,7 @@ export class PackageCommand {
         await packageService.batchExportPackages(packageKeys, includeDependencies);
     }
 
-    public async batchImportPackages(spaceMappings: string[], exportedDatapoolsFile: string, exportedPackagesFile: string): Promise<void> {
-        await packageService.batchImportPackages(spaceMappings ?? [], exportedDatapoolsFile, exportedPackagesFile);
+    public async batchImportPackages(spaceMappings: string[], dataModelMapping: string, exportedPackagesFile: string): Promise<void> {
```

Comment:
I think this should be a list of strings

Reply:
There should probably be two options, either providing the output file from the import of dataModels (which can be implemented when that is ready), or you can define the mappings yourself like `--variableMappings dm-variable,dm-1;dm-variable2,dm-2`, etc.